.envrc

```
@@ -0,0 +1,17 @@
+strict_env
+
+VENV_PATH=".venv"
+
+if [ ! -d "${VENV_PATH}" ]; then 
+  python3 -m venv ${VENV_PATH}
+fi
+
+source ${VENV_PATH}/bin/activate
+
+if [ ! -f "./.direnv" ]; then
+  pip install -e .
```

```suggestion
  pip install --editable .
```
Try to use long options where possible in your code so that it's easier to read for people unfamiliar with the tool!